src/test/java/org/interledger/InterledgerAddressTest.java

```diff
@@ -70,7 +82,7 @@ public void test_address_with_blank() {
    * Validates adding an address prefix (as an InterledgerAddress) to an address prefix.
    */
   @Test
-  public void test_address_prefix_with_address_prefix() {
```

Comment:
Don't think we run checkstyle on our unit tests, but I recall it not liking this underscore function naming style, so made these camel-case.